index.js

```diff
@@ -277,7 +278,14 @@ const validators = [
       Object.keys(refs).forEach((id) => validator.addSchema(refs[id], id));
       return validator;
     },
-    test: (validator, json) => validator.validate(json).valid,
+    test: (validator, json) => validator.validate(JSON.parse(json)).valid,
+  },
+  {
+    name: "turbo-json-parse",
+    setup: (schema) => {
+      return turboJsonParse(schema);
+    },
+    test: (validator, json) => validator(json),
```

Comment:
@mafintosh When I do this test of running the JSON schema testsuite against tupbo-json-parse, I get an out of memory exception in node.js
```
<--- Last few GCs --->

[10081:0x102a83000]    17282 ms: Mark-sweep 2045.1 (2056.4) -> 2045.5 (2052.4) MB, 270.0 / 0.0 ms  (+ 315.6 ms in 65 steps since start of marking, biggest step 8.1 ms, walltime since start of marking 589 ms) (average mu = 0.076, current mu = 0.017) alloca

<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x10097cc39]
    1: StubFrame [pc: 0x100903ac1]
Security context: 0x3d5cd5fc08d1 <JSObject>
    2: parse [0x3d5c7e82c8e1] [0x3d5c977c04b1 <undefined>:~1] [pc=0x188eafc0f8b5](this=0x3d5c942be0d1 <JSGlobal Object>,0x3d5c7e82b3a1 <String[29]: [[[1],[2],[3]],[[4],[5],[6]]]>,29)
    3: arguments adaptor frame: 1->2
    4: test [0x3d5c72912031] [/Users/allanebdrup/Dropbox/json-schema-benchmark/index.js:288] [bytecode=0x3d5...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

Writing Node.js report to file: report.20200510.082656.10081.0.001.json
Node.js report completed
 1: 0x1010285f9 node::Abort() (.cold.1) [/usr/local/bin/node]
 2: 0x10008634d node::FatalError(char const*, char const*) [/usr/local/bin/node]
 3: 0x10008648e node::OnFatalError(char const*, char const*) [/usr/local/bin/node]
 4: 0x100187c07 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]
 5: 0x100187ba7 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/usr/local/bin/node]
 6: 0x100315955 v8::internal::Heap::FatalProcessOutOfMemory(char const*) [/usr/local/bin/node]
 7: 0x1003171ca v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [/usr/local/bin/node]
 8: 0x100313bfc v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/usr/local/bin/node]
 9: 0x1003119fe v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/usr/local/bin/node]
10: 0x10031d8ca v8::internal::Heap::AllocateRawWithLightRetry(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/usr/local/bin/node]
11: 0x10031d951 v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [/usr/local/bin/node]
12: 0x1002eb58a v8::internal::Factory::NewFillerObject(int, bool, v8::internal::AllocationType, v8::internal::AllocationOrigin) [/usr/local/bin/node]
13: 0x10063e698 v8::internal::Runtime_AllocateInYoungGeneration(int, unsigned long*, v8::internal::Isolate*) [/usr/local/bin/node]
14: 0x10097cc39 Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_NoBuiltinExit [/usr/local/bin/node]
Abort trap: 6
```

Can turbo-json-parse not be used in this way?